### ISSUE TITLE
Attempt to reproduce the results of rnn-ext + RL in the paper

### ISSUE THREAD

**Issue by kenchan0226** [ChenRocks/fast_abs_rl#46]
Hi, I tried to reproduce the results of rnn-ext + RL (without abstractor).
First, I pre-train an extractor using train_extractor_ml and construct the proxy training labels using ROUGE-L F1 score. Then, I run the train_full_rl.py and I do not specify the path of abstractor. After I finish the training, the results are significantly lower than the values reported in your paper. I just wonder if it is incorrect to use ROUGE-L F1 score to construct the proxy labels for rnn-ext, or you used other reward functions for it? Thank you so much for your help!

**Comment by ChenRocks** [comment]
I used the same proxy label in ML training stage so you don't need to create an F1 version. In RL training stage, if abstractor is not specified the code simply make abstractor return the extracted sentences. The reward is still R-L (F1). Let me know whether you can reproduce, thanks!

**Comment by kenchan0226** [comment]
Finally, I can reproduce the results when I use the same proxy label R-L( recall) as yours. Thank you so much!
